### ISSUE TITLE
wildcard 

### ISSUE THREAD

**Issue by ghost** [galeone/letsencrypt-lighttpd#8]
Should we update this script to use wildcard certs?

**Comment by galeone** [comment]
It would be cool.  Is the wildcard certs nowadays 100% supported by letsencrypt? If so, this addition is more than welcome

**Comment by ghost** [comment]
Yes, but there is one problem at Debian/Ubuntu. The cerbot ist not yet ready for renew. You must create every 90 days a new TXT record at your NS to get new certs.
So it is nothing for a renew script. But maybe a option on start.


5. Sep. 2018, 11:33 von notifications@github.com:

**Comment by galeone** [comment]
Why not, if it's something that can be automatized why not adding this feature

**Comment by ghost** [comment]
Yeah, but a script could not set the TXT records.


-- 
Liebe Grüße

Malte Kiefer



5. Sep. 2018, 12:22 von notifications@github.com:
